src/ZendCodingStandard/ruleset.xml

```diff
@@ -201,232 +70,823 @@
+         the beginning or at the end of the line, not a mix of both. -->
+    <!-- checked by PSR12.ControlStructures.ControlStructureSpacing -->
+    <!-- checked by PSR12.ControlStructures.BooleanOperatorPlacement -->
+    <!-- checked by Squiz.ControlStructures.ControlSignature -->
+
+    <!-- Additional Zend Framework rules -->
+
+    <!-- The `continue` control structure MAY NOT be used in switch statements,
+         `break` SHOULD be used instead. -->
+    <rule ref="WebimpressCodingStandard.ControlStructures.ContinueInSwitch"/>
+
+    <!-- 5.3.1 while -->
+
+    <!-- Expressions in parentheses MAY be split across multiple lines, where
+         each subsequent line is indented at least once. When doing so, the
+         first condition MUST be on the next line. The closing parenthesis and
+         opening brace MUST be placed together on their own line with one space
+         between them. Boolean operators between conditions MUST always be at
+         the beginning or at the end of the line, not a mix of both. -->
+    <!-- checked by PSR12.ControlStructures.ControlStructureSpacing -->
+    <!-- checked by PSR12.ControlStructures.BooleanOperatorPlacement -->
+    <!-- checked by Squiz.ControlStructures.ControlSignature -->
+
+    <!-- 5.3.2 do while -->
+
+    <!-- Expressions in parentheses MAY be split across multiple lines, where
+         each subsequent line is indented at least once. When doing so, the
+         first condition MUST be on the next line. Boolean operators between
+         conditions MUST always be at the beginning or at the end of the line,
+         not a mix of both. -->
+    <!-- checked by PSR12.ControlStructures.ControlStructureSpacing -->
+    <!-- checked by PSR12.ControlStructures.BooleanOperatorPlacement -->
+    <!-- checked by Squiz.ControlStructures.ControlSignature -->
+
+    <!-- 5.4 for -->
+
+    <!-- Expressions in parentheses MAY be split across multiple lines, where
+         each subsequent line is indented at least once. When doing so, the
+         first expression MUST be on the next line. The closing parenthesis and
+         opening brace MUST be placed together on their own line with one space
+         between them. -->
+    <!-- checked by PSR12.ControlStructures.ControlStructureSpacing -->
+    <!-- checked by PSR12.ControlStructures.BooleanOperatorPlacement -->
+    <!-- checked by Squiz.ControlStructures.ControlSignature -->
+
+    <!-- TODO: allow multiline for structure -->
+
+    <!-- 5.5 foreach -->
+
+    <!-- exclude these messages as they are already checked by PSR2.ControlStructures.ControlStructureSpacing -->
+    <!-- checked by Squiz.ControlStructures.ForEachLoopDeclaration -->
+
+    <!-- 5.6 try, catch, finally -->
+
+    <!-- Additional Zend Framework rules -->
+
+    <!-- All catch blocks MUST be reachable. -->
+    <rule ref="SlevomatCodingStandard.Exceptions.DeadCatch"/>
+
+    <!-- Catch blocks SHOULD catch `Throwable` instead of `Exception`. -->
```

Comment:
I'm unsure about this one.

A `Throwable` can reference a PHP error, while an `Exception` can only reference exceptions. Users should generally only try to catch things that libraries are throwing, not errors generated by the language or an extension.

Reply:
Understood. Removed the rule.